src/modules/constants/constants.controller.ts

```diff
@@ -4,7 +4,6 @@ import { ApiBearerAuth, ApiOperation, ApiParam, ApiResponse, ApiTags } from '@ne
 import { ConstantsService } from './constants.service';
 import { GetConstantsSpiQueryDto } from './dto/get-constants-spi-query.dto';
 import { ConstantSpiEntity } from './entities/constants-spi.entity';
-
 @ApiBearerAuth()
```

Comment:
I'm surprised we don't have a blank line between the last import and our code here

Reply:
nope - it's ESLint formatting. I'll double check

Reply:
this is updated now